spec/support/vcr.rb

```diff
@@ -0,0 +1,19 @@
+# spec/support/vcr.rb
+
+# VCR.configure do |c|
```

Comment:
Not using VCR, so leaving these files commented out. However, I might choose to implement in the future, so leaving in the app in case.